src/components/TodoContent.tsx

```diff
@@ -0,0 +1,244 @@
+        setTodosFromServer(currentTodos =>
+          currentTodos.filter(todo => !todo.isOptimistic),
+        );
+        setErrorMessage(Errors.notAdd);
+        setInputDisabled(false);
+        setTimeout(resetErrorMessage, 3000);
+      })
+      .finally(() => {
+        setLoadingTodo(null);
+      });
+  }
+
+  function setLoadingValue(todo: Todo | null) {
+    setLoadingTodo(todo);
+  }
+
+  useEffect(() => {
+    getTodos()
+      .then(data => {
+        setErrorMessage('');
+        setTodosFromServer(data);
+      })
+      .catch(() => {
+        setErrorMessage(Errors.notLoad);
+        setTimeout(() => {
+          setErrorMessage('');
+        }, 3000);
+      });
+  }, []);
+
+  useEffect(() => {
+    if (todosFromServer.every(todo => todo.completed)) {
+      setIsAllCompleted(true);
+    } else {
+      setIsAllCompleted(false);
+    }
+
+    setCompletedTodos([...todosFromServer].filter(todo => todo.completed));
+  }, [todosFromServer]);
+
+  useEffect(() => {
+    if (todoFieldRef.current) {
+      todoFieldRef.current.focus();
+    }
+  }, [todosFromServer, errorMessage]);
+
+  return (
+    <>
+      <div className="todoapp__content">
+        <Header
+          todosFromServer={todosFromServer}
+          isAllcompleted={isAllcompleted}
+          handleAllButtonClick={handleAllButtonClick}
+          handleSubmit={handleSubmit}
+          todoFieldRef={todoFieldRef}
+          inputDisabled={inputDisabled}
+          todoValue={todoValue}
+          handleInputChange={handleInputChange}
+        />
+
+        <TodoList
+          todos={filterTodosByStatus()}
+          completedTodos={completedTodos}
+          loadingTodo={loadingTodo}
+          anyLoading={anyLoading}
+          clearCompleteLoading={complTodoDeleteLoading}
+          setTodos={handleUpdateTodos}
+          deleteTodo={handleDeleteTodo}
+          setErrorMessage={handleErrorMessage}
+          resetError={resetErrorMessage}
+          setLoadingTodo={setLoadingValue}
+        />
+
+        {todosFromServer.length ? (
+          <Footer
+            handleFilterStatus={status => setFilterStatus(status)}
+            active={active}
+            filterStatus={filterStatus}
+            completedTodos={completedTodos}
+            handleClearCompleted={handleClearCompleted}
+          />
+        ) : null}
+      </div>
+
+      <div
+        data-cy="ErrorNotification"
+        className={classNames(
+          'notification is-danger is-light has-text-weight-normal',
+          {
+            hidden: errorMessage === '',
+          },
+        )}
+      >
+        <button data-cy="HideErrorButton" type="button" className="delete" />
+        {errorMessage === Errors.notLoad && Errors.notLoad}
+        {errorMessage === Errors.notEmpty && Errors.notEmpty}
+        {errorMessage === Errors.notAdd && Errors.notAdd}
+        {errorMessage === Errors.notDelete && Errors.notDelete}
+        {errorMessage === Errors.notUpdate && Errors.notUpdate}
```

Comment:
```suggestion
        {errorMessage}
```
can we just write like this?

Reply:
Я чуть не плакав від сміху, коли це побачив